### DISCUSSION TITLE
How to get rid of the links in the console tab?

### DISCUSSION THREAD

**Discussion by GeVanCo** [gitextensions/gitextensions#11666] [Q&A]
I recently moved to version 4.2.1 of Git Extensions and one of the changes I saw is that the files shown in the console window are now treated as links.
Example:
When I run a `git status` command then I see all my modified files.  But when moving the mouse pointer over the files then the files get an underline and the mouse pointer changes into a "hand" shape giving the impression it's a link (which it seems to be anyway).
I'm not in favour of this behaviour for 2 reasons:

- when clicking the link goes to "nowhere" (I always get a ConEmuC64 error telling me it can't create process with error code 2)
- in the past, when double-clicking on the file it automatically copied the whole path which was very convenient to copy/past the content.  Now, that's not possible anymore.

I couldn't find a way to switch off this behaviour in the settings menu.

Question: is there a way to switch off this behaviour and go back to the way it was before and if so, how?

Environment info:

- Git Extensions 4.2.1.17611
- Build b0c0b284876b63c52c77195ada82a85bfadbb083
- Git 2.44.0.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.28
- DPI 120dpi (125% scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.28 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 7.0.4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by pmiossec** [comment]
Introduced by https://github.com/gitextensions/conemu-inside/pull/34
/cc @gerhardol 

Key modifier to highlight hyperlinks has been set to none otherwise the hyperlinks are not visible/discoverable.
It seems there is no intermediate solution in ConEmu where the hyperlink will be highlighted but made it clickable, with hand cursor, **only** when pressing modifier `ctrl` key (behavior that I find good in VSCode) /cc @Maximus5
![clickable_links](https://github.com/gitextensions/gitextensions/assets/460196/0ddb7f58-e34c-4043-80cc-5f7b3af39f4b)

**Reply by GeVanCo** [reply]
What I "discovered" just now is that when you do Ctrl-double-Click then you're able to select the line (just like it was before).  Else, the "link" appears.
Would have been better to do it the other way around but hey, I can live with that too!

**Comment by gerhardol** [comment]
The click to open link feature was very hidden before, no user I talked to knew about it, so it was set to default so it could be discovered.
So what pmiossec suggest should be done to to change the defaults.
The ConEmu Far Manager should be able to open the link though.

The workaround to mark the link should not be be too hard.
